main/iio-sensor-proxy/files/iio-sensor-proxy

```diff
@@ -0,0 +1,5 @@
+# iio-sensor-proxy service
+
+type = process
+command = /usr/libexec/iio-sensor-proxy
+depends-on = dbus
```

Comment:
is this enough? i think you also need local.target but no idea

Reply:
it's technically enough because you get the local.target dependency through dbus, but is always good to be explicit anyway

Reply:
the PR currently contains my locally working setup yes

Reply:
ah, sure, just saw elogind had local.target on itself too